### PR TITLE
[AKS] Add a hello world test

### PR DESCRIPTION
**Pull request by arompa**
Added a test calling the RootController to check if the app can connect with the db.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,6 +129,8 @@ by executing the following command:
   docker-compose up
 ```
 
+// The port should be 2704, shouldn't it?
```
Comment (zielinskikamil) [inline README.md]: should be even different as 2703 and 2704 are used by api ;/

File: config/pmd/ruleset.xml [hunk 2]
```diff
@@ -1,75 +1,107 @@
...
+          class="net.sourceforge.pmd.lang.rule.XPathRule">
+        <description>
+            RestAssured tests should have assertions either using 'assert...' or 'then'
+        </description>
+        <priority>3</priority>
+        <properties>
+            <property name="xpath">
+                <value>
+                    <![CDATA[
+//ClassOrInterfaceType[
```
Comment (zielinskikamil) [inline config/pmd/ruleset.xml]: is it needed? or can we format it as least?

File: config/pmd/ruleset.xml [hunk 3]
```diff
@@ -1,75 +1,107 @@
...
+@Image='Test'
+]/../../../MethodDeclaration[
+not(descendant::*[(starts-with(@Image, 'assert') or
+starts-with(@Image, 'Assert') or starts-with(@Image, 'then'))])
+]
+]]>
+                </value>
+            </property>
+        </properties>
+        <!--<example><![CDATA[]]></example>-->
```
Comment (zielinskikamil) [inline config/pmd/ruleset.xml]: this line can be removed

File: config/pmd/ruleset.xml (hunk 2 above)
Comment (arompa) [reply config/pmd/ruleset.xml]: Yes - this is needed. That's how the custom rule is defined

**Pull request merged**